src/utils/garden-utils.js

```diff
@@ -0,0 +1,7 @@
+import daoList from '@1hive/gardens-dao-list'
+import { addressesEqual } from './web3-utils'
+
+export function getGardenLabel(address) {
```

Comment:
Initially i used `name` but the changed it to `label` because the function returns the `address` of the garden if there's no `name` associated with the garden. And i don't think `title` is accurate in this case. Why do you think `label` is not the right one here?